### ISSUE TITLE
model can't adapt to different intrinsics?

### ISSUE THREAD

**Issue by wytalfred** [nv-tlabs/lift-splat-shoot#21]
Is it true that one model can only use one set of intrinsics? when I try to run your pre-trained model on my own dataset which has different intrinsics, object distances become slightly wrong.

**Comment by manueldiaz96** [comment]
Sorry if this is a dumb question, but do you use your own intrinsics?
For sure both the Categorical Depth layers and BevEncode parts of the model will be affected by different intrinsics, since these affect directly the projection step.

**Comment by VeeranjaneyuluToka** [comment]
I am wondering why changing intrinsics/extrinsics effect the performance as they are being used to convert from camera to BEV space and which is not learning based. Is projection step is not a part of this converion?

**Comment by manueldiaz96** [comment]
You are right, but my point was more about the fact that intrinsics (and to a lesser extent extrinsics too) may affect the distribution in the Top-down view on which the data was trained.
